src/electron/views/screenshot/screenshot-view.tsx

```diff
@@ -2,26 +2,35 @@
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
 import { ScreenshotContainer } from 'electron/views/screenshot/screenshot-container';
+import { isEmpty } from 'lodash';
 import * as React from 'react';
 import { ScreenshotViewModel } from './screenshot-view-model';
+import * as styles from './screenshot-view.scss';
 
 export type ScreenshotViewProps = { viewModel: ScreenshotViewModel };
 
 export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', (props: ScreenshotViewProps) => {
+    const isUnavailable = isEmpty(props.viewModel.screenshotData);
+
     /* note the h1 below has a tab index because the screenshot image is scrollable and keyboard users need to be able to
     navigate via tab per https://dequeuniversity.com/rules/axe/3.3/scrollable-region-focusable */
     return (
-        <div>
+        <div className={styles.screenshotView}>
             <h1 tabIndex={0}>Target Page Screenshot</h1>
-            <ScreenshotContainer
-                screenshotData={props.viewModel.screenshotData}
-                highlightBoxRectangles={props.viewModel.highlightBoxRectangles}
-            />
-            {renderDeviceNameCaption(props.viewModel.deviceName)}
+            {isUnavailable ? renderUnavailableMessage() : renderScreenshotContainer(props.viewModel)}
         </div>
     );
 });
 
-function renderDeviceNameCaption(deviceName?: string): JSX.Element {
```

Comment:
I'm guessing this (removing the device name) was updated on the figma mock. Just checking this wasn't remove inadvertently.

Reply:
Correct, apparently it confused several users in user testing into thinking that the thing labelled "Android Emulator" was going to be an actual interactive emulator, so the caption was removed to reduce confusion